packages/shopify-cart/src/graphql/fragments/merchandise.ts

```diff
@@ -1,5 +1,6 @@
 export default () => /* GraphQL */ `
   fragment Merchandise_merchandise on ProductVariant {
+    sourceEntryId: id
```

Comment:
We can get rid of this because it's already defined as a required field in the `merchandise` query in `src/graphql/fragments/cart.ts`.

```suggestion
```